pulpcore/pulpcore/app/models/publication.py

```diff
@@ -107,21 +107,36 @@ def delete(self, **kwargs):
             CreatedResource.objects.filter(object_id=self.pk).delete()
             super().delete(**kwargs)
 
+    def update_distributions(self, model=None):
```

Comment:
In case of Docker, where we have a DockerDistribution, this method will need to be called explicitly after a publication is created. The doc string should make that clear.

Reply:
Done.